sbin/Release.sh

```diff
@@ -13,84 +13,40 @@
 # limitations under the License.
 #
 
-TIMESTAMP="$(date +'%Y%m%d')"
 npm install
-for f in OpenJDK*.tar.gz
+# Expect a naming convention for build artifacts to follow this pattern:
+# OpenJDK<version>_<arch>_<os>_<timestampOrTag>.<extension>
+# Examples: OpenJDK8_x64_Windows_201813060547.zip, 
+#           OpenJDK8_x64_LinuxLH_201813060547.tar.gz, 
+#           OpenJDK10_aarch64_Linux_201813060547.sha256.txt
+#           OpenJDKamber_x64_Linux_201813061304.tar.gz	
+for f in OpenJDK*
```

Comment:
Minor nitpick, what is a 'f'? :-)

Reply:
f is the variable some other person used, but agree, will update to 'file'  ;)